processor/src/test/java/org/mapstruct/ap/test/selection/qualifier/handwritten/Titles.java

```diff
@@ -22,15 +22,16 @@
 
 import org.mapstruct.ap.test.selection.qualifier.annotation.EnglishToGerman;
 import org.mapstruct.ap.test.selection.qualifier.annotation.TitleTranslator;
-//import org.mapstruct.ap.test.selection.qualifier.annotation.TitleTranslator;
 
 import com.google.common.collect.ImmutableMap;
+import org.mapstruct.Named;
 
 /**
  *
  * @author Sjaak Derksen
  */
 @TitleTranslator
+@Named( "TitleTranslator" )
```

Comment:
What does it mean if the qualifier is given on the type level? Just seeing it for `@TitleTranslator`, too, btw. I always had it in mind for methods only. Is it the default for all methods declared by this type?